composer.json

```diff
@@ -18,7 +18,7 @@
         "php": "^7.3 || ^8.0",
         "silverstripe/framework": "^4.10",
         "silverstripe/cms": "^4.1",
-        "symbiote/silverstripe-queuedjobs": "^4",
+        "symbiote/silverstripe-queuedjobs": "^4.9",
```

Comment:
Why is this increased?

Reply:
Fix prefer-lowest build
https://github.com/creative-commoners/silverstripe-ldap/runs/7191398040?check_suite_focus=true#step:10:687
```
ERROR [Emergency]: Uncaught Error: Call to undefined method Symbiote\QueuedJobs\Services\QueuedJobService::singleton()
[2022-07-05 07:02:02] error-log.ERROR: Uncaught Exception Error: "Call to undefined method Symbiote\QueuedJobs\Services\QueuedJobService::singleton()" at /home/runner/work/silverstripe-ldap/silverstripe-ldap/vendor/silverstripe/session-manager/src/Jobs/GarbageCollectionJob.php line 65 {"exception":"[object] (Error: Error: Call to undefined method Symbiote\\QueuedJobs\\Services\\QueuedJobService::singleton() 
```

Reply:
`QueuedJobService` is `Injectable` as of `4.1.0`.
Please use `^4.1` as the constraint.

Reply:
As [noted](https://github.com/silverstripe/silverstripe-session-manager/pull/112#discussion_r922905438) - phpunit 9 compatibility